certora/specs/EnumerableMap.spec

```diff
@@ -69,13 +69,13 @@ invariant atUniqueness(uint256 index1, uint256 index2)
 ┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
 │ Invariant: index <> value relationship is consistent                                                                │
 │                                                                                                                     │
-│ Note that the two consistencyXxx invariants, put together, prove that at_ and _indexOf are inverse of one another.  │
+│ Note the two consistencyXxx invariants, put together, prove that at_ and _positionOf are inverse of one another.    │
```

Comment:
We should either do 
- "Note that ..." as in "Please note that ..."
- "Note: ..."

But the current formulation is not correct